CHANGELOG.next.asciidoc

```diff
@@ -188,6 +188,8 @@ automatic splitting at root level, if root level element is an array. {pull}3415
 - Collect missing remote_cluster in elasticsearch ccr metricset {pull}34957[34957]
 - Add context with timeout in AWS API calls {pull}35425[35425]
 - Fix no error logs displayed in CloudWatch EC2, RDS and SQS metadata {issue}34985[34985] {pull}35035[35035]
+- Fix statsd module's behaviour of immediate stopping of parsing further metrics after encountering any invalid packet {pull}35075[35075]
```

Comment:
Can we use a shorter message like "Improve statsd module parsing for invalid packets"?

Reply:
I'll try but "Fix statsd module parsing for invalid packets" seems a little off as we are not fixing the parsing of invalid packets but the behavioural of the statsd module when invalid packets are encountered.

Here are a few suggestions that are shorter than the actual changelog:

1. Fix immediate stopping of statsd metrics parsing when encountering an invalid packet.
2. Resolve the issue with statsd module's prematurely halting metric parsing upon encountering an invalid packet.
3. Resolve statsd module's prematurely halting of metrics parsing upon encountering an invalid packet.

If these suggestions also look a bit lengthy or off, let me know. I'll think of more.

Reply:
It's just a suggestion, feel free to decide. The PR link provides detailed issue/fix description anyway.

Reply:
Going for 3rd then.